scipy/datasets/_funcs.py

```diff
@@ -0,0 +1,192 @@
+"""
+Functions that load datasets.
```

Comment:
In the meantime (moving toward 1.2.0) this would be a good place to try to distill a summary of the scope of this submodule from the mailing list.